.github/workflows/rspec_tests.yml

```diff
@@ -0,0 +1,82 @@
+name: Rspec tests
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  rspec_tests:
+    name: ${{ matrix.cfg.os }}(ruby ${{ matrix.cfg.ruby }})
+    strategy:
+      matrix:
+        cfg:
+          - {os: ubuntu-18.04, ruby: 2.3}
```

Comment:
did you wanna remove this?

Reply:
oh sorry, this PR should be closed, #1099 has the updates we would want.